### DISCUSSION TITLE
Dictionary Directory while using "add yo dictionary" option - Debian 12 bookworm

### DISCUSSION THREAD

**Discussion by moustaleurie** [vkbo/novelWriter#2005] [Help] [answered]
Hello, I am writing because I made a mistake and added to the dictionary a word accidentally. So, now when it is written wrong, it shows up as fine. I searched all aspell and hunspell directories and ".dic" , ".aff" files I could find. I also searched the ~/.config/enchant directory. There I can find a file with saved words from earlier, but the specific word I added today was not included. I also reinstalled aspell and hunspell. I also searched the home directory and the /usr directory for files that included the word. Nothing... Do you have any advice? Thank you in advise!

**Comment by vkbo** [comment, marked as answer]
Go to the menu to Tools > Project Word List.

**Reply by vkbo** [reply]
The words are saved in `meta/userdict.json` in your projects by the way. They have nothing to do with the spell checker library itself.

**Reply by moustaleurie** [reply]
Thank you very much. I read about novelWriter using system's dictionary and enchant and it never crossed my mind personal dictionaries are saved per project. I am sorry for your time, I feel I should have find out on my own... Thank you very much once again, for your help and for novelWriter in general! 
Take care
